stratum/hal/lib/common/p4_service_test.cc

```diff
@@ -157,6 +157,8 @@ class P4ServiceTest : public ::testing::TestWithParam<OperationMode> {
       metadata_id: 666666
       value: "\x12"
   )";
+  static constexpr char kTestPacketMetadata4[] = R"(
+  )";
```

Comment:
can an empty string `""` work?

Reply:
I'm sure it would, but this form keeps the code consistent with the above lines.